language_detector/main.py

```diff
@@ -2,10 +2,24 @@
 
 """This is the entry point of the program."""
 
-from languages import LANGUAGES
+from .languages import LANGUAGES
+
+def detectInOneLanguage(text,lang):
```

Comment:
Agree with above, including logic involving the frequency of the common words (such as highest count) would better ensure an accurate result